tests/rest/client/test_redactions.py

```diff
@@ -569,50 +571,81 @@ def test_redact_relations_txn_id_reuse(self) -> None:
         self.assertIn("body", event_dict["content"], event_dict)
         self.assertEqual("I'm in a thread!", event_dict["content"]["body"])
 
-    def test_content_redaction(self) -> None:
-        """MSC2174 moved the redacts property to the content."""
+    @parameterized.expand(
+        [
+            # Tuples of:
+            #   Room version
+            #   Boolean: True if the redaction event content should include the event ID.
+            #   Boolean: true if the resulting redaction event is expected to include the
+            #            event ID in the content.
+            (RoomVersions.V10, False, False),
+            (RoomVersions.V11, True, True),
+            (RoomVersions.V11, False, True),
```

Comment:
It took me a moment to see that the first bool controls what the test does, and the second bool defines what the test asserts. (I thought they were both controlling assertions, which didn't make sense given that V11 appears twice.)

Reply:
Do you have a suggestion on how to make it clearer?

Reply:
I think it's fine; any action is on me to read more carefully.